views/settings.blade.php

```diff
@@ -47,7 +58,19 @@
                     </header>
                     <div class="panel-body">
                         <span class="amount reg-list-link"><a id="recruitment_link"
-                                                              href='{{ url('/join/' . $recruitment_key) }}'>{{ url('/join/' . $recruitment_key) }}</a></span>
+                                                              href='{{ url('/join/' . $recruitment_key .'/stockist') }}'>{{ url('/join/' . $recruitment_key .'/stockist') }}</a></span>
+                    </div>
+                @endif
+
+                @if($auth->dropship && $recruitment_dropship_key)
```

Comment:
Server side, do we check the user is able to refer, before allowing them to refer?